bitbybit/README.md

```diff
@@ -39,7 +39,8 @@ How this works:
 
 - #[bitfield(u32)] specifies that this is a bitfield in which u32 is the underlying data type. This means that all the
   bits inside of the bitfield
-  have to fit within 32 bits. Built-in Rust types (u8, u16, u32, u64, u128) as well as arbitrary-ints (u17, u48 etc) are supported.
```

Comment:
Not sure why Clion felt the need to reformat this, but shouldn't make a difference